documentation/1.11/content/userguide/tools/create-aux-image.md

```diff
@@ -24,7 +24,7 @@ Usage: imagetool createAuxImage [OPTIONS]
 | `--tag` | (Required) Tag for the final build image. Example: `store/oracle/mydomain:1`  |   |
 | `--additionalBuildCommands` | Path to a file with additional build commands. For more details, see [Additional information](#--additionalbuildcommands). |
 | `--additionalBuildFiles` | Additional files that are required by your `additionalBuildCommands`.  A comma separated list of files that should be copied to the build context. See [Additional information](#--additionalbuildfiles). |
-| `--builder`, `-b` | Executable to process the Dockerfile. Use the full path of the executable if not on your path. | `docker`  |
+| `--builder`, `-b` | Executable to process the Dockerfile. Use the full path of the executable if not on your path. | Defaults to 'docker', or, when set, to the value in environment variable WLSIMG_BUILDER. |
```

Comment:
WLSIMG_BUILDER -> `WLSIMG_BUILDER` (env variable should be in code font; please correct globally)

Reply:
Done. Also changed 'docker' to `docker`.